docs/source/algorithms/Abins2D-v1.rst

```diff
@@ -0,0 +1,107 @@
+.. algorithm::
+
+Credits
+-------
+
+Author:
+    Adam Jackson
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+Abins2D is a plugin for Mantid which allows scientists to generate theoretical inelastic neutron scattering spectra (INS) in 2-D :math:`(|\mathbf{q}|,\omega)` space, accounting for the limitations of real-world measurements.
+
+The intensity is calculated in the "almost-isotropic" incoherent powder-averaging approximation, as used in the 1-D Abins algorithm.
+Different 2-D INS instruments may be selected, along with instrument parameters (incident energy, chopper settings) that determine the accessible :math:`(|\mathbf{q}|,\omega)` region and energy resolution.
+More information about the implemented working equations can be found :ref:`here <DynamicalStructureFactorFromAbInitio>`.
+
+Abins requires data from existing lattice dynamics calculations to determine the phonon mode frequencies and displacements. These are used to compute INS intensities; higher-order phonons may approximated as combinations of fundamental modes.
+Several file formats are supported for frequency and displacement data at given :math:`\mathbf{q}`-points: CASTEP (.phonon), CRYSTAL (.out), GAUSSIAN (.log), DMOL3 (.outmol) or VASP (.xml).
+For VASP XML inputs, it is also permitted to use a "selective dynamics" in which some atoms are frozen.
+These would typically form a rigid substrate (e.g. a noble metal surface) for an adsorbed molecule of lightweight atoms, and are omitted from the calculated spectrum.
```

Comment:
Is this only true for the selective dynamics?

Reply:
It is really, rigid surfaces absolutely do vibrate. The approximation only works if you're able to neglect the intensity of the substrate vibrations, or they are active in a different frequency range from the adsorbate. Both of which are fair assumptions if you are looking at stretching modes of hydrogenous molecules.

Reply:
could we make it clearer in the docs? I was unsure when reading it

Reply:
Abins will accept your frozen atoms regardless of whether they are a good physical approximation.

I guess I can be more explicit that you probably shouldn't do this in other cases?